README.md

```diff
@@ -10,7 +10,7 @@ Building
 make all
 ```
 
-The project uses essentially a copy and paste of the Makefile from the [regular input plugin for mupen64plus](https://github.com/mupen64plus/mupen64plus-input-sdl) With the dynamic config lib and SDL dependencies removed.
+The project uses essentially a copy and paste of the Makefile from the [regular input plugin for mupen64plus](https://github.com/mupen64plus/mupen64plus-input-sdl) With the dynamic config lib and SDL dependencies removed. It includes flags to link libjson-c, which it assumes has been installed in /usr/local/lib and /usr/local/include (you can get this library [here](https://github.com/json-c/json-c)).
```

Comment:
Out of curiosity did you build this on your system or install via a package manager? It looks like its available on ubuntu as `libjson-c2`. Can you add a note about the Ubuntu package name if this is the case